content/docs/user-guide/managing-external-data.md

```diff
@@ -1,7 +1,7 @@
 # Managing External Data
 
-> ⚠️ Artifacts added with --external are not affected by dvc push/pull/status
-> -c. You are likely looking for
+> ⚠️ Artifacts added with --external are not affected by
```

Comment:
`are not affected by "dvc push/pull/status -c"` - @efiop  what would be the explicit way to explain this to someone less technical/DVC proficient?

Reply:
@shcheklein Not sure what you mean.

Reply:
I guess my point here is that this note is so specific and technical that I would not expect it being understood by users exploring different ways to deal with data with DVC.

Could you explain here in a comment, not using any DVC terms, what do we want to convey with this disclaimer? I'll try to help to find a better way to phrase this.

Reply:
I mean that this note should be probably written like this (if I understand its meaning correctly):

Note, the way to handle data described below can be considered an advanced alternative to an actual `dvc add`, `dvc push` workflow. In some cases you do need it indeed, in other cases it might be you have a better, simpler solution. If, for example, you are looking for a way to add data to DVC without moving it to the machine first, you might check this option [straight-to-remote/cache](https://github.com/iterative/dvc/issues/4520), or `dvc import-url` ...

Otherwise my point that it's too low level, it will be very hard for folks exploring different ways of managing data with DVC to understand what implications does `are not affected by dvc push/pull/status -c` have, why is that a problem.

They way it is phrased now, it feels very abrupt be at the very beginning - no context, no explanation.

Reply:
The aim was to put a short and quick disclaimer to address the typical questions we get. I might need to get back to this later for a verbose explanation.

Reply:
Yep, I think we are on the same page on this. I see all this questions and understand what you are trying to achieve. My only concern that they way it's written it won't move a needle since people won't understand this. People don't have enough context when they come here. For example, they might easily think that "advanced" is for them (why not, right?). If we don't know how to explain it better, I'm fine - let's merge it as is, may be it'll change it a bit indeed.

Reply:
Leaving as is to save time for now.